src/main/java/io/jenkins/plugins/appcenter/utils/Utils.java

```diff
@@ -0,0 +1,26 @@
+package io.jenkins.plugins.appcenter.utils;
+
+import hudson.model.Describable;
+import hudson.model.Descriptor;
+import io.jenkins.plugins.appcenter.Messages;
+import jenkins.model.Jenkins;
+
+public class Utils {
```

Comment:
To be honest, I don't really understand why need this file or these helper methods. Was there a use case you had in mind?